src/modules/0.0.22/aws_codepipeline/mappers/pipeline_declaration.ts

```diff
@@ -201,8 +212,9 @@ export class PipelineDeclarationMapper extends MapperBase<PipelineDeclaration> {
       return out;
```

Comment:
cannot comment in the exact line, but `cloud.update` need to be updated. `const cloudRecord = ctx?.memo?.cloud?.PipelineDeclaration?.[pd.name ?? ''];` is not true anymore, we need to get the cloudRecord using the `this.entityId(e)`